Build/Customizations/SqlServer/WindowsSqlServerDockerImage.cs

```diff
@@ -0,0 +1,57 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Containers;
+using DotNet.Testcontainers.Networks;
+using JetBrains.Annotations;
+using Serilog;
+
+namespace Customizations.SqlServer;
+
+public class WindowsSqlServerDockerImage : ISqlDockerImage
+{
+  private readonly IContainer _container;
+  private readonly string _connectionString;
+
+  public WindowsSqlServerDockerImage (string imageName, string name, [CanBeNull] INetwork network)
+  {
+    const string password = "P@ssw0rd";
+    var containerBuilder = new ContainerBuilder()
+        .WithImage(imageName)
+        .WithHostname(name)
+        .WithName(name)
+        .WithEnvironment("SaPassword", password)
+        .WithEnvironment("MinSqlServerMemory", "0")
+        .WithEnvironment("MaxSqlServerMemory", "8192")
+        .WithCreateParameterModifier(e => e.HostConfig.Isolation = "hyperv");
+
+    if (network != null)
+      containerBuilder = containerBuilder.WithNetwork(network);
+
+    _container = containerBuilder.Build();
+
+    Log.Information($"Starting SQL Server using image '{imageName}'.");
+    _container.StartAsync().GetAwaiter().GetResult();
+
+    var properties = new Dictionary<string, string>
+                     {
+                         { "Server", name },
+                         { "Initial Catalog", "master" },
+                         { "User Id", "sa" },
+                         { "Password", password },
+                         { "TrustServerCertificate", bool.TrueString }
+                     };
+    _connectionString = string.Join(";", properties.Select(property => string.Join("=", property.Key, property.Value)));
```

Comment:
.net contains a connectionstring builder
While using string.join is technically robust enough, it feels like we're re-inventing the wheel
https://learn.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlconnectionstringbuilder